### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by carloso0114**
In this branch, I create the basic HTML structure, set up webpack and linters following the gitflow flow.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+<body class="d-flex flex-column min-vh-100">
+  <header>
+    <h1 class="m-5">Leaderboard</h1>
+  </header>
+  <div class="container">
+    <div class="row">
+      <div class="col">
+        <div class="d-flex justify-content-between py-5">
+          <h2>Recent Scores</h2>
+          <button type="button" class="btn btn-light">Refresh</button>
```
Comment (abruzy) [inline src/index.html]: Good job

- [x] Kindly remove the duplicate, you have the same content in both your main and the one in the dist folder, your main index.html should contain only one root element.
Screenshot 2021-07-27 at 23 09 07

**Review by abruzy — CHANGES_REQUESTED** [review]
```markdown
Hi @carloso0114 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.html (hunk 1 above)
Comment (carloso0114) [reply src/index.html:26]: I'm sorry but that's not a requirement.
image

Comment (carloso0114) [reply src/index.html]: thanks, I guess you meant the dist/index.html file to be in production mode.

**Review by abruzy — APPROVED** [review]
```markdown
Hi @carloso0114 👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**